client/test/integration/index.ts

```diff
@@ -2,38 +2,48 @@ import * as path from 'path'
 import * as Mocha from 'mocha'
 import * as glob from 'glob'
 import { IntegrationTestHelper } from './markLogicIntegrationTestHelper'
+import * as vscode from 'vscode'
 
-export function run(): Promise<void> {
+export async function run(): Promise<void> {
     const mocha = new Mocha({
         ui: 'tdd'
     })
     globalThis.integrationTestHelper = new IntegrationTestHelper()
+    await globalThis.integrationTestHelper.beforeEverything()
+
+    // Without extending the setup, only 1 reporter is support.
+    // For available reporters, see https://mochajs.org/#reporters
+    // For options on configuring multiple reports, see:
+    // https://github.com/mochajs/mocha/pull/1360
+    // https://github.com/glenjamin/mocha-multi
+    const resultsFileName = __dirname + '/../../../results/integrationTestResults.xml'
+    mocha.reporter('xunit', { output: resultsFileName })
+    // mocha.reporter('spec')
+
     mocha.options.color = true
     mocha.rootHooks({
-        beforeAll: async () => {
-            await globalThis.integrationTestHelper.beforeEverything()
-        },
         beforeEach: () => {
             globalThis.integrationTestHelper.setupEachTest()
         },
-        afterAll: async () => {
-            await globalThis.integrationTestHelper.afterEverything()
-        },
         afterEach: () => {
             globalThis.integrationTestHelper.teardownEachTest()
         }
     })
+    // afterAll: async () => {
```

Comment:
Can this be deleted instead of commented out?

Reply:
It can be. I'm just not happy with it yet. I'd really like to delete the test files, but the delete function isn't reliable. So, my plan is to come back to this sometime soon.